### PR TITLE
AA-352 Improve Sort UX in Automation Calculator

### PR DESCRIPTION
**Pull request by kialam**
Spoke with @trahman73 on this and the fix is to:

1. Dynamiclaly show in the UI on the right side Templates list column the sort by details:
sortBy_ROI_templates_list

2. Include all the sort by attributes/values returned by the API in the popover like so:
Screen Shot 2021-02-15 at 8 17 10 PM

### CODE REVIEW

**Comment by benthomasson** [comment]
https://issues.redhat.com/browse/AA-352

File: src/Containers/AutomationCalculator/TopTemplates.js [hunk 1]
```diff
@@ -76,98 +73,93 @@ const InputAndText = styled.div`
...
+    sortBy = null,
     setDataRunTime = () => {},
     setEnabled = () => {},
     redirectToJobExplorer = () => {}
 }) => (
     <Card style={{ overflow: 'auto', flex: '1 1 0' }} className="top-templates">
         <CardBody>
             <p>Enter the time it takes to run the following templates manually.</p>
-            {data.map(({
-                id,
```
Comment (brumik) [inline src/Containers/AutomationCalculator/TopTemplates.js:121]: Did you do this by accident? I am pretty sure I just destructured in a recent commit this object on your advice.

**Review by brumik — CHANGES_REQUESTED** [review]
Also if we merge the #416 first we could pass `sortAttr` directly instead of `sortBy`.

Except for the default value of the `sortBy` everything else is a nitpick (non-blockers).

File: src/Containers/AutomationCalculator/TopTemplates.js (hunk 1 above)
Comment (kialam) [reply src/Containers/AutomationCalculator/TopTemplates.js:121]: No, this is intentional. We are now passing in the entire api response to show in the tooltip so it makes less sense to destructure everything out in this case.

**Review by brumik — APPROVED** [review]
Thanks for including my nitpicks ^^ :heart:

**Pull request merged**